README.md

```diff
@@ -913,6 +913,11 @@ See ```VULKAN_HPP_EXPECTED```.
 
 With this define you can include a reflection mechanism on the vk-structures. It adds a function ```reflect``` that returns a tuple-version of the structure. That tuple then could easily be iterated. But at least for now, that feature takes lots of compile-time resources, so currently it is recommended to enable that feature only if you're willing to pay that price.
 
+#### VULKAN_HPP_NO_WIN32_PROTOTYPES
+
+By default, if ```DYNAMIC_LOADER_TOOL``` is enabled on Win32, ```vulkan.hpp``` declares ```HANDLE```, ```LoadLibraryA```, and other required symbols. It could cause conflicts with the ```Windows.h``` alternatives, such as ```WindowsHModular```.

```

Comment:
Maybe use the complete term `VULKAN_HPP_ENABLE_DYNAMIC_LOADER_TOOL` here, instead of just `DYNAMIC_LOADER_TOOL`.
Also, `HANDLE` is not declared here, but `HINSTANCE`.

Reply:
Done, I assume